### PR TITLE
Change 'ticks' to 'SF ticks'

### PR DESCRIPTION
**Pull request by Phoenix-Starlight**

### CODE REVIEW

**Comment by svr333** [comment]
why is every one of your prs 545435 commits?

**Comment by Phoenix-Starlight** [comment]
Uhhhh due to the squashing it will always be that whenever I pull from upstream it will be my commits + 1. So. The only option would be to destroy the fork and refork.

**Comment by WalshyDev** [comment]
Or just... pull the changes.

**Comment by Phoenix-Starlight** [comment]
Actually, let me see if rollbacking to a place without my commits keeps it intact or not..

**Comment by Phoenix-Starlight** [comment]
Pulling doesn't work because it says its up to date.

**Comment by WalshyDev** [comment]
Then you aren't pulling from the right branch/origin

**Comment by Phoenix-Starlight** [comment]
Ok, sorted out commit mess and all.

**Review by svr333 — DISMISSED** [review]

File: pages/Server-Optimizations.md [hunk 1]
```diff
@@ -90,18 +90,18 @@ options:
...
+You can configure this setting in your `plugins/Slimefun/config.yml`. We recommend to only make small changes to the default value of 10 ticks when necessary.
 ```yaml
 URID:
-  custom-ticker-delay: 12
+  custom-ticker-delay: 10
 ```
 
 Similar to this setting, Slimefun regularly checks Player's armor to apply effects that come with wearing specific sets of armor.
-The default setting for this task is set to 10 ticks (20 ticks = 1 second).<br>
+The default setting for this task is set to 10 SF ticks (20 SF ticks = 1 second).<br>
```
Comment (WalshyDev) [inline pages/Server-Optimizations.md:104]: This is correct as MC ticks

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: pages/Server-Optimizations.md (hunk 1 above)
Comment (Phoenix-Starlight) [reply pages/Server-Optimizations.md:104]: Oh, oops!

**Review by svr333 — APPROVED** [review]

**Review by WalshyDev — APPROVED** [review]

**Pull request merged**